store/src/main/java/org/apache/rocketmq/store/CommitLog.java

```diff
@@ -569,7 +586,19 @@ public PutMessageResult putMessage(final MessageExtBrokerInner msg) {
             }
         }
 
-        long elapsedTimeInLock = 0;
+
+        InetSocketAddress bornSocketAddress = (InetSocketAddress) msg.getBornHost();
+        if (bornSocketAddress.getAddress() instanceof Inet6Address) {
+            msg.setBornHostV6Flag();
+        }
+
+        InetSocketAddress storeSocketAddress = (InetSocketAddress) msg.getStoreHost();
+        if (storeSocketAddress.getAddress() instanceof Inet6Address) {
+            msg.setStoreHostAddressV6Flag();
+        }
+
+        long eclipseTimeInLock = 0;
```

Comment:
The ```eclipseTimeInLock``` should be ```elapsedTimeInLock```?  And there are several places have the same wrong naming, please correct all of them.

Reply:
What you said has been revised.